source/light/contracts/Light.sol

```diff
@@ -238,31 +210,27 @@ contract Light is ILight, Ownable {
     }
 
     // Transfer token from sender to signer
-    IERC20(senderToken).safeTransferFrom(
+    IERC1155(senderToken).safeTransferFrom(
```

Comment:
See note about about rentranchy guard, Since we are using safeTransferFrom I believe do we want to add on non reentrant to this function https://docs.openzeppelin.com/contracts/4.x/api/security

Reply:
ok cool.  Will add.

Reply:
Left a comment on another PR, we may not need this on these since it is going from sender to signer, I was more looking at safeTransferFrom as something we should be implementing on these types of calls but it may require a little more digging.